docs/SignedAPKAndroid.md

```diff
@@ -78,6 +78,8 @@ Simply run the following in a terminal:
 $ cd android && ./gradlew assembleRelease
 ```
 
+> If you prefer you can just run `react-native run-android --configuration=release` that the **release** `apk` will be generated and installed into the connected `android` device.
```

Comment:
Not sure if this is grammatically correct, `that the **release** `apk``, shouldn't be `that will generate the release apk`?